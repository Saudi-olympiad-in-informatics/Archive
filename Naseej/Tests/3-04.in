36319 379
12554 14318 268240486
17111 34880 288838655
27644 545 595641733
1397 29186 986737046
13415 20786 210021357
15961 35864 644848232
32677 20272 840391166
9210 16110 364050496
13243 2381 262442221
34183 28242 404475887
12944 29172 624057650
7052 20177 701394321
3921 33495 644613476
5680 30177 849729651
32269 11031 206279814
33117 23354 456019885
25842 20001 327859581
954 9876 400311503
13740 11610 424415364
33122 20508 34820835
4843 25752 551540889
24709 35945 158942733
11311 4436 491272091
9585 2564 816752008
27469 14684 772107122
3790 14792 352858482
11738 18388 684179352
23612 10105 769879731
14134 33305 371012654
11001 22539 292362826
35658 26037 679436463
29530 2705 971562059
26813 16800 759955224
36227 2431 841742952
32672 27851 70837760
21394 24864 401040282
10376 18719 914821035
13391 18802 637737194
30325 32060 799987888
15172 26185 861762810
28081 30687 258088660
33515 1587 583038723
34278 2848 114532400
29515 1608 51992748
20807 20582 916333602
27660 19875 274196984
17944 32602 407587927
12386 22610 77325132
1004 13263 564699690
26779 32121 347557117
32614 13846 653246865
7160 24990 776337205
13851 23746 314056592
33823 10397 823049272
4829 17651 875131121
6688 26588 840976109
16650 12802 887560189
18050 7080 754374434
34073 31254 411211323
17746 4832 917496843
4833 25262 969230975
31838 22355 412600969
3729 1234 988934822
23277 3583 658284254
11069 18896 780149142
7022 9806 282399743
8154 11688 242528873
20433 13575 365749766
7632 28145 941551021
9892 17295 765435954
29686 9713 460943444
23020 12051 794958815
17749 28759 843721526
16686 5411 277256639
26473 19887 419585918
26651 1712 69265221
20264 5617 619393272
10962 15062 927043510
8592 15697 805149800
29967 18556 149488738
17595 5654 183270016
4684 22444 547646445
31278 30061 797589145
29768 31512 203730136
15181 25175 589116655
16709 27677 712380739
29677 1584 630697877
6752 17852 124241655
25931 17979 820766090
13279 26783 710228234
23008 14153 244016329
6004 22125 174771517
22642 16367 768817990
5996 15554 631626518
33556 26757 381797854
15901 18689 163404821
22503 8324 330042649
6670 16831 738767438
29723 7226 305594743
1999 31999 315893894
30520 4308 290585304
5141 18039 640242447
18797 180 642442039
7990 21528 969942184
26910 18028 584424660
31564 9018 543993310
4385 20959 257477457
23892 35927 718815545
32304 25776 702455866
12311 13486 262449545
12495 30429 580810574
5656 33687 759633208
482 18865 885198226
36267 24134 360953812
34317 34895 429092584
4109 29716 530451881
19624 3579 77933757
18324 18490 149581686
14746 33677 588466853
2146 15230 760976948
6725 161 787348520
13168 30423 141469255
26010 34303 425081620
20925 12887 125233709
8818 26952 957511192
2529 13534 479132290
28604 16913 962944909
20403 9138 327037050
33681 5480 17050228
29124 23058 816402081
32547 30733 555986968
17025 11690 203036736
17579 31612 79731364
16747 19443 769917075
29553 1105 987622072
9836 12437 400485828
3025 31577 960531403
35280 15906 700527361
7518 23452 158207231
20644 27361 719083309
31733 1106 321179253
29094 6284 348953947
11701 11438 923507157
15421 7924 362419024
34648 1864 401726778
22796 8202 701009249
104 12242 87712347
13656 3027 184810952
517 11035 581191390
16023 20465 200136669
21509 13440 215436083
14631 33072 318637936
30848 22804 460564694
31339 8768 697196552
12179 12667 332576031
2351 1220 824759662
20999 29825 531540099
10842 21559 676642540
6273 5183 464058369
5155 12004 585195232
26922 5985 810279609
15471 14824 389060162
2997 31771 33978390
27674 33671 667463724
21939 24599 122713235
27790 33335 251451758
2620 10061 684942585
34228 18800 47367342
442 4389 458474116
34102 6681 611858717
9485 10504 13623632
15390 13479 85270449
9233 33900 266614614
34826 13912 574400833
30933 24344 224975220
33091 23228 543285835
606 15157 199720431
11767 17646 123896983
29652 17404 592882961
31086 6362 916243840
19783 30590 869149846
4964 26490 288063183
7420 10008 506065925
35921 36199 294296636
14850 13861 875072705
2196 27465 917239622
6372 29159 896987273
35819 17591 52212653
3847 31877 419633231
33882 35531 290962451
19896 8266 959405468
33974 7621 302001219
28874 35197 997796568
13136 28353 10425074
12188 2530 493711232
23995 15194 679704564
1368 27246 867270824
9704 5537 224379198
13670 796 355467893
5395 33820 395871052
5126 11932 4503899
19930 30319 503008557
3415 23095 328112587
30303 22014 555510776
12314 8797 984816090
14240 27525 281560100
26380 9404 383448525
14207 11909 481899325
7720 10357 264207896
12691 33791 349123285
33148 29003 56814465
12236 32394 911903357
26556 22172 613227996
32281 1017 718197499
16730 32388 775377155
2442 30385 560909169
1009 25239 958146335
6043 33099 952759336
4086 4455 168257399
2429 19057 915862169
10752 25448 698890134
23563 18998 684726615
16177 35246 498636346
29437 16603 601163964
13411 6714 574292167
13653 9394 166280973
2948 8194 645558999
17515 27835 571518843
33659 3474 815880145
8394 22152 930267052
6110 2733 283817578
31290 31565 640785517
14095 14480 274538044
25490 21006 917743309
4313 24749 304446479
29275 34915 74251807
18158 8922 943656615
19159 17321 407925059
19627 18921 809993131
4441 16635 23184920
2506 20262 220750474
11444 21227 160413021
8119 26924 95721694
15034 3143 95393656
14827 8970 390924345
12300 32948 410561349
23684 21631 390566163
33788 4679 923866975
9890 27841 349729732
1486 21897 770107794
25796 27194 515362950
27985 3200 898280422
29285 21482 489838069
6493 11071 818930715
34480 21503 529857441
17348 1869 8369206
35104 33569 383847769
24525 19238 830508585
17481 5034 576241124
8261 15433 501326890
19670 30225 777000848
2940 12603 926311894
19913 36093 702418943
5083 28780 332647980
15717 2906 900159383
16849 14722 557752528
9946 12474 612718993
8653 30008 458576171
5604 3326 443848918
9613 6328 917413259
33978 32789 41726836
12452 4549 463119421
2316 12647 936656415
10209 24229 350367971
33748 7539 601155951
2594 9338 542558509
26855 3627 566638646
32518 17350 664882487
23032 14378 138744225
27817 15480 610524247
28905 35045 52867906
55 13222 504709527
26155 25486 248909389
29661 540 655938779
29176 15305 227579785
16273 6324 820315548
5553 10907 191689516
17039 10489 743896126
32086 20587 764638039
27236 33167 435257236
30081 33998 799257589
13827 19651 243538570
34865 19663 273156909
12333 24428 466357770
15776 25138 339260981
34802 30991 891392640
16365 30715 800705403
18581 14127 483994761
28728 35534 876109610
20716 13011 931783551
22474 24936 304121635
14876 29969 976262478
7790 8708 430631313
10184 16754 711811053
15974 25187 814984834
29274 13404 799376558
32988 6002 644015481
25345 26320 412140070
1797 23752 699884932
25888 14902 234203593
8801 33060 103685764
2836 32926 997725549
19843 34842 896739394
15875 28072 574296550
20238 10031 505585915
6018 8901 294336453
4591 6575 122971906
15943 6381 476803732
34560 26940 397944570
28056 2736 899991484
34636 13838 382395660
4760 23923 409840227
27010 20060 956782319
2804 17926 200378685
24368 27613 427823528
6451 1812 997807212
36176 8842 37883137
15898 29466 49490750
17219 23382 57824907
6342 31814 747945523
10264 1038 676287993
12706 30891 795458017
33053 19832 529434506
27001 33552 756387598
138 20067 963900942
29533 25846 275484620
2132 20858 113058985
3519 30034 69631410
16696 29547 37179016
24747 25561 601892977
20905 11251 968029673
29242 19975 629180761
22746 16729 735397879
22466 3156 63663930
12074 8157 130252601
15568 1875 737851639
32010 25368 6287718
28579 13752 236011029
3081 12951 585259588
26613 15676 372146928
15069 23601 549997796
36039 32740 711327262
19872 1843 901157745
26851 16222 600124176
23928 21591 88753958
27793 20459 660676111
29075 28571 137329724
15452 31287 574196647
8885 26877 24709064
19897 33472 399130080
22748 2325 445947200
29896 15031 520592537
17847 6884 830989616
31598 22218 75860955
32504 254 726713904
33036 7132 597061565
22031 7273 910295900
7356 6613 406189244
32514 21232 323512218
6616 34120 200460698
11150 11178 180339031
15805 34011 6851664
6453 32076 314159692
19130 18344 106317104
23620 26496 327957173
15834 1588 491094148
27721 17414 844819239
24824 12651 466583171
3046 17137 782271013
