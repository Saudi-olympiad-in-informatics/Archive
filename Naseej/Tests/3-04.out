13560991357988